src/components/HighlightedText/__test__/HighlightedText.spec.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import styled from 'styled-components';
+import HighlightedText from '../';
+
+describe('<HighlightedText />', () => {
+    const parts = [
+        {
+            value: 'Apples',
+            type: 'text',
+        },
+        {
+            value: 'varieties',
+            type: 'hit',
+        },
+        {
+            value: 'Honeycrisp',
+            type: 'text',
+        },
+    ];
+    it('should return 3 <span> with theirs respective values because the default wrapp ers are span', () => {
+        const component = mount(<HighlightedText parts={parts} />);
+        const container = component.find('span');
+        expect(
+            container
+                .at(0)
+                .html()
+                .includes('Apples'),
+        ).toBe(true);
+        expect(
+            container
+                .at(1)
+                .html()
+                .includes('varieties'),
+        ).toBe(true);
+        expect(
+            container
+                .at(2)
+                .html()
+                .includes('Honeycrisp'),
+        ).toBe(true);
+    });
+    it('should return 3 <p> with theirs respective values because the custom wrappers are p', () => {
+        const TextContainer = styled.p`
+            color: grey;
+        `;
+
+        const HitContainer = styled.p`
+            color: #fff;
+        `;
+
+        const component = mount(
+            <HighlightedText
+                parts={parts}
+                textComponent={TextContainer}
+                hitComponent={HitContainer}
+            />,
+        );
+        const container = component.find('p');
+        expect(
+            container
+                .at(0)
+                .html()
+                .includes('Apples'),
+        ).toBe(true);
+        expect(
+            container
+                .at(1)
+                .html()
+                .includes('varieties'),
+        ).toBe(true);
+        expect(
+            container
+                .at(2)
+                .html()
+                .includes('Honeycrisp'),
+        ).toBe(true);
+    });
```

Comment:
we should change these tests to make clearer the difference between a hit and normal text

Reply:
this second test you can delete the first the third test all the code